runtime/src/integration/mod.rs

```diff
@@ -223,6 +191,13 @@ pub async fn with_timeout<T: Future>(future: T, duration: Duration) -> T::Output
     timeout(duration, future).await.expect("timeout")
 }
 
+pub async fn start_chain() -> std::io::Result<Child> {
+    let command = Command::new("sh").arg("../scripts/run_parachain_node.sh").spawn();
```

Comment:
Is it possible to keep the parachain running and refactor the tests to support using existing state?

Reply:
Even with `instant-seal` it takes some time to startup which we could avoid.

Reply:
I have researched it, and the solution is 
- To set up Chain Storage to the `Genesis state` for every test case. However, this approach has its own limitations as any changes in runtime business logic need to update the `set_genesis_state` method on the client side accordingly.